src/workloads/execution/PingCommand.yml

```diff
@@ -0,0 +1,49 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/product-perf"
+Description: >
+  This is a simple test that runs ping command on MongoDB to meassure the latency of command dispatch.
+
+Clients:
+  Default:
+    QueryOptions:
+      maxPoolSize: 500
+
+MaxPhases: &MaxPhases 1
+ActorTemplates:
+- TemplateName: PingTemplate
+  Config:
+    Name: {^Parameter: {Name: "Name", Default: "TwoThreadPing"}}
+    Type: RunCommand
+    Threads: {^Parameter: {Name: "Threads", Default: 1}}
+    Phases:
+      OnlyActiveInPhases:
+        Active: {^Parameter: {Name: "Active", Default: [0]}}
+        NopInPhasesUpTo: *MaxPhases
+        PhaseConfig:
+          Duration: {^Parameter: {Name: "Duration", Default: "30 seconds"}}
+          GlobalRate: {^Parameter: {Name: "Rate", Default: "100 per 1 second"}}
+          Database: admin
+          Operations:
+          - OperationName: RunCommand
+            OperationIsQuiet: true
+            OperationCommand:
+              ping: 1
+
+Actors:
+- ActorFromTemplate:
+    TemplateName: PingTemplate
+    TemplateParameters:
+      Name: TwoThreadsPingRateLimited
+- ActorFromTemplate:
```

Comment:
I would consider putting a quiesce between these 2 phases, functionally it won't really do anything but it will allow the internal fsync to show where one phase stopped and the next started.

This isn't hugely significant here as the jump in the number of connections will also show the change so feel free to ignore this.